packages/model/src/research-output.ts

```diff
@@ -128,6 +136,19 @@ export const researchOutputMapType = (
   return null;
 };
 
+export const researchOutputMapPublishingEntity = (
+  publishingEntity?: string | null,
+): ResearchOutputPublishingEntities | null => {
+  if (
+    publishingEntity &&
+    (publishingEntity === 'Team' || publishingEntity === 'Working Group')
+  ) {
+    return publishingEntity;
+  }
+
+  return null;
```

Comment:
I wonder if we should throw. This seems like it introduce complexities for cases we shouldn't have